README.md

```diff
@@ -209,6 +212,7 @@ The `input` value defined for your policy will resemble the JSON below:
 
 ```json
 {
+  "parsed_path": ["api", "v1", "products"],
```

Comment:
It's worth documenting this field a little bit more since everything else in the input comes from the standard Envoy ext_authz check request type. I'd explicitly call this field out and mention when/how to use it.

Reply:
Done